torch/onnx/verification.py

```diff
@@ -122,6 +122,7 @@ def _compare_ort_pytorch_outputs(
     atol: float,
     check_shape: bool,
     check_dtype: bool,
+    ignore_none: bool,
```

Comment:
If optional, I would annotate as Optional[bool]. Same is needed for rtol and atol.

Reply:
Tested with lintrunner, and found these can't be optional as they are consumed later in the code.